.commitlintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["@commitlint/config-conventional"]
```

Comment:
Helps conform commits to a common standard https://commitlint.js.org/. I've been manually adopting this in our OSS projects because it helps to keep changelogs consistent, this just formalizes that here.